src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+
+import 'bootstrap/scss/bootstrap.scss';
+
+import './assets/scss/index.scss';
+import App from './app/App';
+
+render(
+    <BrowserRouter>
+        <App />
+    </BrowserRouter>,
+    document.getElementById('reactApp'),
```

Comment:
I think DOM values (like element ID here) shouldn't be in camelCase.

Reply:
Done. camelCase was removed from  element with id reactApp. Now it has id react-app.